src/tests/specs/workItemTableTest.spec.ts

```diff
@@ -112,4 +112,11 @@ describe('Work Item datatable list: ', () => {
     await browser.sleep(3000);
     expect(await planner.workItemList.hasWorkItem(c.newWorkItem1.title)).toBeTruthy();
   })
+
+  it('clicking on label should filter the workitem list by label', async() => {
+    await planner.workItemList.clickWorkItemLabel(c.workItemTitle7);
+    expect(await planner.header.getFilterConditions()).toContain('label: '+c.label2);
```

Comment:
This is not how you would compare the value. It should be
`expect(await planner.header.getFilterConditions()).toContain(c.label2);`

Reply:
@divyanshiGupta Instead of doing a `..toContain('label: ' + c.label2)` you could sanitize the array in the `getFilterConditions` function so that we have `...toContain(c.label2)`.

Reply:
@jarifibrahim I can sanitize the array and remove 'label: ' from each string but then it adds a layer of complexity and extra calculations. Do you think its worth all the effort just to get a simpler check statement in the test? Also, we have a few tests in other components where we assert for labels like this only with extra meta data attached to the label name. That is why i think this approach is okay.

Reply:
@divyanshiGupta we are asserting like this only for 1 test which is calendar date test. So, in Iteration calendar test, We are taking value based on current month and year. So, we need to follow this approach. But, for this test you can use in a simple way. WDYT?